CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Fixes
 - Fix endMedia to not be removed from DOM on mouseleave for `ListItem` @musingh1 ([#278](https://github.com/stardust-ui/react/pull/278))
+- Make `secondary` and `primary` props mutually exclusive for `Button` @layershifter ([#429](https://github.com/stardust-ui/react/pull/429))
```

Comment:
I would omit this in changelog as the *Unreleased version* is the one which introduces #419.